chia/rpc/rpc_server.py

```diff
@@ -147,6 +147,7 @@ class RpcServer:
     ssl_context: SSLContext
     ssl_client_context: SSLContext
     webserver: Optional[WebServer] = None
+    daemon_heartbeat: int = 300
```

Comment:
Is this default repetition useful?  I'm honestly not entirely sure.  Maybe we can say that this form isn't right and either no parameter default or a centralized variable used everywhere we set the default would be appropriate?  Maybe.  Huh, or even a load of the default config file and using the values there...  a step too far for this PR though at least.

Reply:
I can see that changing the default would require changing it in several code places - so perhaps some cleanup isn't a bad idea

Reply:
Would having a `default_heartbeat = 300` variable and using it as the default in the other two places be good or bad?

Reply:
I think it would be mostly a wash on that.